frontend/src/components/forms/CreateResident.tsx

```diff
@@ -0,0 +1,254 @@
+import React, { useEffect, useState } from "react";
```

Comment:
make sure to run the linter, I'm getting a lot of eslint warnings

Reply:
Th linter has been run 🙋‍♂️

Reply:
these are fixed [here](https://github.com/uwblueprint/supportive-housing/pull/114)